ex2_header_css.css

```diff
@@ -0,0 +1,40 @@
+.header {
```

Comment:
If there is sufficient overlap between the two, I'd much rather have a common file with the common code and, if needed, separate files with the styling that is specific per page. As soon as you notice yourself duplicating code, stop and think about ways of making the code reusable instead of copying and pasting stuff in multiple places. That is an approach prone to human error, especially on large projects or in large teams, where not everybody knows what everybody else is doing / has done. Subtle differences slip through the cracks, and before you know it, one part of your application starts looking considerably different than the others. There's a primordial principle of good software development to keep in mind: "Don't Repeat Yourself" - aka the DRY principle.